airbyte-integrations/connectors/destination-mysql-strict-encrypt/build.gradle

```diff
@@ -28,6 +28,16 @@ dependencies {
 
     implementation 'mysql:mysql-connector-java:8.0.22'
 
+    // TODO: declare typing-deduping as a CDK feature instead of importing from source.
+    implementation project(':airbyte-cdk:java:airbyte-cdk:typing-deduping')
+    integrationTestJavaImplementation testFixtures(project(':airbyte-cdk:java:airbyte-cdk:typing-deduping'))
+
     integrationTestJavaImplementation project(':airbyte-integrations:connectors:destination-mysql')
     integrationTestJavaImplementation libs.testcontainers.mysql
 }
+
+configurations.all {
+    resolutionStrategy {
+        force libs.jooq
+    }
+}
```

Comment:
We need to somehow pull it into some base module and fix the dependency chain instead of forcing in all destinations

Reply:
yeah. I forget _why_ we need this, tbh. isn't the point that we use the version in deps.toml everywhere already?